File: lib-user/src/main/java/top/xinsin/pojo/Route.java
package top.xinsin.pojo;

import lombok.Data;

/**
 * @author xinsin
 * Created On 2022/12/24 16:03
 * @version 1.0
 */
@Data
public class Route {
    private Integer id;
    private String path;
    private String name;
    private String parentId;
    private String component;
    private String title;
    private String auth;
    private Integer status;
    private String remark;
    private String createId;
    private String createTime;
    private String updateId;
    private String updateTime;
}

File: web/src/entities/Route.ts
export interface Route{
    path: string;
    name: string;
    parentId: number | string;
    component: string;
    title: string;
    auth: string;
    status: number | boolean;
    remark: string;
    updateId: number;
    createId: number;
    createTime: string;
    updateTime: string;
    id: number;
}